src/ansys/geometry/core/designer/component.py

```diff
@@ -684,6 +685,73 @@ def create_sphere(self, name: str, center: Point3D, radius: Distance) -> Body:
         self._master_component.part.bodies.append(tb)
         return Body(response.id, response.name, self, tb)
 
+    @protect_grpc
+    @check_input_types
+    @ensure_design_is_active
+    @min_backend_version(24, 2, 0)
+    def create_body_from_loft_profile(
+        self,
+        name: str,
+        profiles: List[List[TrimmedCurve]],
```

Comment:
Why does it have to be a list of lists? Can't It just be a list of TrimmedCurve?

Reply:
It creates body from a list profiles, which is a list of trimmed curves.